draft-ietf-elegy-rfc8989bis.md

```diff
@@ -186,14 +189,16 @@ an attacker must produce 365 volunteers to exceed a 50% chance of NomCom
 capture (see {{capture-math}}).
 
 A sudden surge in the number of volunteers, particularly of people that no one
-recognizes as a part of the community is an early-warning system for leadership
-and the IETF Secretariat to further investigate.
+recognizes as a part of the community, is an early-warning system for leadership
+and the IETF Secretariat to further investigate. One response would be to deny
+fee waivers to suspect attendees, as these are issued at the discretion of the
+IETF and IRTF chairs.
```

Comment:
In the remote-fee draft, it says: "If the free option requires additional registration steps, such as applying for a fee waiver, those requirements should be clearly documented." An application suggests a possible refusal? Section 4 of that draft also mentions potential misuse. I'll instead refer to Section 4 rather than pinning it on the chairs.

Reply:
I think the intent is that remote fee waivers be available freely, i.e., without much checking or process, and that they only be limited in extraordinary circumstances. So this draft should assume they are widely available to anyone.